src/nodes/Conditional.js

```diff
@@ -9,12 +9,8 @@ const Conditional = {
     group(
       concat([
         path.call(print, 'condition'),
-        indent(line),
-        '? ',
-        path.call(print, 'trueExpression'),
-        indent(line),
-        ': ',
-        path.call(print, 'falseExpression')
+        indent(concat([line, '? ', path.call(print, 'trueExpression')])),
```

Comment:
Does it make sense to do `[line, '?', line, ...]` here?

Reply:
it does. Thank you for spotting this